app/src/main/java/com/monkeyteam/chimpagne/ui/HomeScreen.kt

```diff
@@ -206,115 +200,58 @@ fun HomeScreen(
                   uiState.currentUserProfilePicture,
                   onClick = {
                     if (!accountViewModel.isUserLoggedIn()) {
-                      showPromptLogin = true
+                      promptLogin(context, navObject)
                     } else {
                       navObject.navigateTo(Route.ACCOUNT_SETTINGS_SCREEN)
                     }
                   },
                   modifier = Modifier.testTag("account_settings_button"))
             })
+      },
+      floatingActionButton = {
+        ExtendedFloatingActionButton(
+            onClick = { navObject.navigateTo(Route.EVENT_CREATION_SCREEN) },
+            icon = { Icon(Icons.Filled.Add, "organize an event") },
+            text = { Text(stringResource(R.string.homescreen_organize_event)) },
```

Comment:
I personally think floating action buttons should not have text on them, plus "organize an event" takes way to much screen space (which makes it ugly):
<img width="200" alt="image" src="https://github.com/Chimpagne/ChimpagneApp/assets/125994939/ba4e36af-1c83-4472-886b-4e04c2ad206e">
So I am personally for the removal of this text field on this floating action button, just like for how you did on the figma:
<img width="137" alt="image" src="https://github.com/Chimpagne/ChimpagneApp/assets/125994939/cddb09c6-e092-4ffe-af15-40675ff91b89">